tests/integration/long_callback/utils.py

```diff
@@ -94,14 +94,19 @@ def setup_long_callback_app(manager_name, app_name):
                 "2",
                 "--loglevel=info",
             ],
+            encoding="utf8",
             preexec_fn=os.setpgrp,
             stderr=subprocess.PIPE,
         )
         # Wait for the worker to be ready, if you cancel before it is ready, the job
         # will still be queued.
+        lines = []
         for line in iter(worker.stderr.readline, ""):
```

Comment:
this stop value was never hit because readline() was returning bytes and b"" != ""